runironic.sh

```diff
@@ -1,4 +1,35 @@
 #!/usr/bin/bash
+PATH=$PATH:/usr/sbin/
+DATADIR="/var/lib/mysql"
+if [ ! -f "$DATADIR/ironic-initalized" ]; then
+    crudini --set /etc/my.conf mysqld max_connects 64
+    crudini --set /etc/my.conf mysqld max_heap_table_size 1M
+    crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
+    crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
+    mysql_install_db --datadir=$DATADIR
+    chown -R mysql /var/log/mariadb
+    chown -R mysql $DATADIR
+    cd /usr
```

Comment:
Why do we need this?

Reply:
1) setup initial db otherwise it does not start
2) Without tyring to interactively execute mysql_secure_installation, we need to ensure that mysql is the owner of the logs and data directory as line 9 was executed as root.
3) The change of the working directory is from the manual instructions on how to start mysql_safe.

Reply:
>     3. The change of the working directory is from the manual instructions on how to start mysql_safe.

That's what I was wondering about. Thanks @juliakreger